android/quest/src/test/resources/configs/ecbis-saa/config_sync.json

```diff
@@ -1,6 +1,14 @@
 {
   "resourceType": "Parameters",
   "parameter": [
+    {
+      "resource": {
+        "resourceType": "SearchParameter",
+        "base": [
+          "Group"
+        ]
+      }
+    },
```

Comment:
Kindly update this to match what's in the server.

Reply:
Sure, will update this, on my next push